prow/test/integration/integration-test.sh

```diff
@@ -35,34 +38,40 @@ function retry() {
 }
 
 function setup() {
-  "${bazel}" run //prow/test/integration:setup-local-registry "$@" || ( echo "FAILED: set up local registry">&2; return 1 )
-
-  if [[ "${PUSH_IMAEG_WITH_KO:-}" == "true" ]]; then
-    go run ./hack/prowimagebuilder --ko-docker-repo="localhost:5001" --prow-images-file="prow/test/integration/prow/.prow-images.yaml" --push
-  else
-    # testimage-push builds images, could fail due to network flakiness
-    (retry "${bazel}" run //prow:testimage-push "$@") || ( echo "FAILED: pushing images">&2; return 1 )
-  fi
+  ./prow/test/integration/setup-local-registry.sh
 
-  "${bazel}" run //prow/test/integration:setup-cluster "$@" || ( echo "FAILED: setup cluster">&2; return 1 )
+  # testimage-push builds images, could fail due to network flakiness
+  retry \
```

Comment:
The `prowimagebuilder` already includes per-image retries. Retrying the entire build process if there is a failure is probably not what we want since it will significantly increase the failure time if there is a legitimate error.

Reply:
good point, done